lib/pages/chat/empty_chat.dart

```diff
@@ -0,0 +1,205 @@
+
+  void setActiveClient(Client c) => setState(() {
+    Matrix.of(context).setActiveClient(c);
+  });
+
+  Future<void> send() async {
+    scrollDown();
+    sendController.value = TextEditingValue(
+      text: sendController.value.text,
+      selection: const TextSelection.collapsed(offset: 0),
+    );
+    
+    _createRoom(onRoomCreatedSuccess: (room) {
+      room.sendTextEvent(
+        sendController.text,
+      );
+    });
+  }
+
+  Future<void> _createRoom({
+    OnRoomCreatedSuccess onRoomCreatedSuccess,
+  }) async {
+    await showFutureLoadingDialog<String>(
+      context: context,
+      future: () => Matrix.of(context).client.startDirectChat(presentationContact.matrixId!)
+    ).then((result) async {
+      if (result.result != null) {
+        final roomId = result.result;
+        if (onRoomCreatedSuccess != null && roomId != null) {
+          final room = Matrix.of(context).client.getRoomById(roomId);
+          if (room != null) {
+           await onRoomCreatedSuccess(room);
+          }
+          VRouter.of(context).toSegments(
+            ['rooms', 'empty', roomId],
+            isReplacement: true,
+          );
+        }
+      }
+    }).onError((error, stackTrace) {
+      Logs().d("_createRoom: $error");
+      VRouter.of(context).pop();
+    });
+  }
+
+  void emojiPickerAction() {
+    if (showEmojiPicker) {
+      inputFocus.requestFocus();
+    } else {
+      inputFocus.unfocus();
+    }
+    emojiPickerType = EmojiPickerType.keyboard;
+    setState(() => showEmojiPicker = !showEmojiPicker);
+  }
+
+  void _inputFocusListener() {
+    if (showEmojiPicker && inputFocus.hasFocus) {
+      emojiPickerType = EmojiPickerType.keyboard;
+      if (mounted) {
+        setState(() => showEmojiPicker = false);
+      }
+    }
+  }
+
+  void scrollDown() {
+    if (scrollController.hasClients) {
+      scrollController.jumpTo(0);
+    }
+  }
+
+  void onEmojiBottomSheetSelected(_, Emoji? emoji) {
+    typeEmoji(emoji);
+    onInputBarChanged(sendController.text);
+  }
+
+  void typeEmoji(Emoji? emoji) {
+    if (emoji == null) return;
+    final text = sendController.text;
+    final selection = sendController.selection;
+    final newText = sendController.text.isEmpty
+        ? emoji.emoji
+        : text.replaceRange(selection.start, selection.end, emoji.emoji);
+    sendController.value = TextEditingValue(
+      text: newText,
+      selection: TextSelection.collapsed(
+        // don't forget an UTF-8 combined emoji might have a length > 1
+        offset: selection.baseOffset + emoji.emoji.length,
+      ),
+    );
+  }
+
+  void emojiPickerBackspace() {
+    sendController
+      ..text = sendController.text.characters.skipLast(1).toString()
+      ..selection = TextSelection.fromPosition(
+        TextPosition(offset: sendController.text.length),
+      );
+  }
+
+  void onInputBarSubmitted(_) {
+    send();
+    FocusScope.of(context).requestFocus(inputFocus);
```

Comment:
- onSubmitted -> close input keyboard, clear focus.

if success, go to the real chat room,
if error, request focus here + toast: create room failed.